terrain/voxel_terrain.cpp

```diff
@@ -27,7 +28,10 @@ VoxelTerrain::VoxelTerrain() {
 	_block_updater = NULL;
 
 	_run_in_editor = false;
-	_smooth_meshing_enabled = false;
+
+	Ref<VoxelLibrary> library;
+	library.instance();
+	set_voxel_library(library);
```

Comment:
Note that following this logic, we could just as well ask Godot to have a tileset by default in `TileMap` and a default MeshLibrary as well in `GridMap`

Reply:
This is just speculation about Godot philosophy in general? I obviously don't know.

I started looking at this with FastNoise. There is the ability to add a recursive lookup. But if you select that option and don't create the sub-object it will crash. So I added the sub-object by default. Similarly, here if you don't add a VoxelLibrary, VT won't display anything. Adding it automatically is one step closer to removing non-display conditions.